views/layout.jade

```diff
@@ -1,11 +1,11 @@
-!!! 5
+doctype html
 html
   head
     title #{settings.sitename} - #{title}
     link(rel='stylesheet', href='/stylesheets/bootstrap.min.css')
-    style(type='text/css')
-      body{padding-top:60px;}
+    link(rel='stylesheet', href='/stylesheets/custom.css')
   body
+  div.upper-header
```

Comment:
See the comment in Custom.css. The Padding top is actually supposed to be in the body of the page, to force the content down below the Navbar which is fixed top, and therefore doesn't need a container with a padding top